### DISCUSSION TITLE
RCT Instanz startet immer wieder neu wenn Wechselrichter aus

### DISCUSSION THREAD

**Discussion by aruttkamp** [aruttkamp/ioBroker.rct#27] [Fragen & Antworten] [answered]
Immer wenn der Wechselrichter aus ist 
z.B. weil kein Speicher und kein PowerSwitch installiert sind 
startet sich RCT immer wieder neu - bis ioBroker dies abbricht und den Adapter stoppt.

**Comment by aruttkamp** [comment]
Lösung für dieses Problem ist es den Adapter kontrolliert zu stoppen ( Blockly ).
Dazu nutze ich den Adapter "Ping" und das Skript im Screenshot. 

![07-08-2023_20-25-12](https://github.com/aruttkamp/ioBroker.rct/assets/1271404/9868f274-47e7-4890-a92f-ed28f95d0b9b)

Die Telegram Ausgaben dienen nur meiner Kontrolle. Die muss man natürlich nicht mit umsetzen.

**Comment by NCIceWolf** [comment, marked as answer]
Das Thema sollte spätestens mit Version >= 1.2.12 gefixt sein.
Der Adapter würde in diesem Fall zwar alle 2 Minuten eine Fehler-Meldung auswerfen, sich dann aber auch wieder verbinden sobald der WR wieder in Betrieb geht.

**Comment by NCIceWolf** [comment]
Da es hier keine Einwände gibt, kann hier wohl auch zu
